lib/Dancer2/Core/Role/Config.pm

```diff
@@ -84,9 +91,11 @@ has global_triggers => (
             },
         };
 
+        no warnings 'once'; # Disable: Name "Dancer2::runner" used only once: possible typo
```

Comment:
It might be worthwhile exploring this warning.

Reply:
Seconded.